### PR TITLE
Added myself

### PR DESCRIPTION
**Pull request by deltaqyto**

### CODE REVIEW

File: names/deltaqyto.txt [hunk 1]
```diff
@@ -0,0 +1,2 @@
+Itsa me
+delta
```
Comment (auscompgeek) [inline names/deltaqyto.txt:2]: *gasp*
the delta?

**Review by auscompgeek — CHANGES_REQUESTED** [review]
lbtm

File: names/deltaqyto.txt (hunk 1 above)
Comment (deltaqyto) [reply names/deltaqyto.txt:2]: It would appear so. What does your change do? "@@ -0,0 +1,2 @@"

**Review by auscompgeek — APPROVED** [review]
you know what good enough

**Pull request merged**